src/modules/UI/scenes/ChangeMiningFee/ChangeMiningFeeExchangeConnector.ui.js

```diff
@@ -1,17 +1,16 @@
 // @flow
 
 import {connect} from 'react-redux'
-import type {State} from '../../../ReduxTypes'
+import type { State, Dispatch } from '../../../ReduxTypes'
 import ChangeMiningFee from './ChangeMiningFee.ui'
-import { changeFee } from '../../../../actions/CryptoExchangeActions'
+import { updateMiningFees } from '../SendConfirmation/action'
```

Comment:
It appears `changeFee` is no longer used. Should remove function and Constants.CHANGE_EXCHANGE_FEE since they aren't used anymore